src/auro-accordion.js

```diff
@@ -142,6 +148,7 @@ export class AuroAccordion extends LitElement {
    * Toggles the visibility of the accordion content.
    */
   toggle() {
+    console.log('button clicked, toggling.......')
```

Comment:
I think we can ditch this console log since it's obvious on the screen that the UI changes.